cerberus/cerberus.py

```diff
@@ -298,6 +316,9 @@ def _validate_definition(self, definition, field, value):
             if validator:
                 validator(definition[rule], field, value)
 
+        if 'rename' in definition:
```

Comment:
`remove_unknown` is included, see https://github.com/misja/cerberus/blob/references/cerberus/cerberus.py#L134